packages/ui/src/components/editor/properties/particle/index.tsx

```diff
@@ -413,7 +413,7 @@ const ParticleSystemNodeEditor: EditorComponentType = (props) => {
         <BooleanInput value={particleSystem.systemParameters.worldSpace} onChange={onSetSystemParm('worldSpace')} />
       </InputGroup>
       <Button className="self-end" onClick={onAddBehavior}>
-        Add Behavior
+        {t('editor:properties.particle-system.world-space')}
```

Comment:
is this the correct text ?, I see you added add behavior into editor.json , I think its intended to be used here